Description: Improve randomness of generated passwords
Origin: upstream, http://bazaar.launchpad.net/~mdeslaur/pasaffe/trunk/revision/573
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/pasaffe/+bug/1309685

Index: pasaffe-0.38/pasaffe/EditDetailsDialog.py
===================================================================
--- pasaffe-0.38.orig/pasaffe/EditDetailsDialog.py	2014-01-23 17:43:36.000000000 -0500
+++ pasaffe-0.38/pasaffe/EditDetailsDialog.py	2014-04-18 14:35:08.493092142 -0400
@@ -80,7 +80,7 @@
 
     def show_passwords_menu(self):
         """Generate some new passwords"""
-        command = ["apg", "-n", "6", "-M", "sNC",
+        command = ["apg", "-a", "1", "-n", "6", "-M", "NCL",
                    "-m", str(self.password_length),
                    "-x", str(self.password_length)]
         try:
